packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/apple/appidpinokta.md

```diff
@@ -0,0 +1,10 @@
+In the **General Settings** section:
+
+* **Name** &mdash; Enter the name that you would expect to see on a button, such as **Sign in with Apple**.
```

Comment:
Is it correct? Can the customer change the text on the button in sign in widget?

Reply:
I don't think the customer can change the text on the button in the sign in widget. The "name" field should be an internal name. cc: @haishengwu-okta to confirm

Reply:
This was my original set up for my Custom Sign-in page and the Sign-in Widget: 
<img width="542" alt="Screen Shot 2020-06-27 at 1 51 39 PM" src="https://user-images.githubusercontent.com/45509385/85930904-5c77cf80-b87d-11ea-8708-9250ddc26d2a.png">

It used to work this way, I thought. I'm assuming there's been changes. I remember being able to change the text that appears....but now am doubting my memory?? :(

However, now that I've tested it again, I do see that it makes no difference any longer -- at least for the social IdPs. I change the text for the 'Name' value and nothing changes on the Sign-In page.

Additionally, you can't just add the 'type' and the 'id' to the html for the Customize Sign-In Page and have the "Sign In with Apple" button appear like the other social identity providers. If I add `{type: 'Apple', id: 'IdP ID'}`, I get this on the button on the Sign-In page:

<img width="470" alt="Screen Shot 2020-06-27 at 2 02 59 PM" src="https://user-images.githubusercontent.com/45509385/85931320-cc3b8980-b880-11ea-81a0-4e01a0889c57.png">

It appears to currently be working like SAML and the Okta to Okta configuration where you have to add `{id: 'idp id', text: 'Sign in with Apple'}` in order for it to have text appear on the button.

<img width="478" alt="Screen Shot 2020-06-27 at 2 03 42 PM" src="https://user-images.githubusercontent.com/45509385/85931328-dc536900-b880-11ea-8186-82222ad1a1a7.png">

Will we have an automatic "Sign in with Apple" button appear for Apple, like we do for Facebook, Google, LinkedIn, and Microsoft?

Also - if it's going to work like SAML and Okta to Okta on the Sign-In page, then I should probably change the information here (add probably add what you have to do differently for SAML and Okta to Okta, too):

Here is the info in the guide on customizing that content: https://developer.okta.com/docs/guides/add-an-external-idp/saml2/use-idp-to-sign-in/

Reply:
That makes sense that you can't change the text for the social IdPs (so that they remain consistent), but can for SAML and Okta to Okta (because those are names specific to the customer). For the Apple IdP button specifically, the text should be defaulted to "Sign in with Apple" and there should be an associated Apple icon. I thought that I saw this reflected in Haisheng's PR, so I wonder where the gap is

Reply:
Zoe is correct. Widget has fixed display name for social buttons (`type` is facebook, google, microsoft, linkedin, apple)

Reply:
I was using a very old widget version. Upgraded and all is well with the Apple IdP button.